soweego/linker/evaluate.py

```diff
@@ -23,9 +15,15 @@
 from pandas import concat
 from sklearn.model_selection import GridSearchCV, StratifiedKFold
 
-from soweego.commons import constants, keys, target_database, utils
+from soweego.commons import constants, target_database, utils
 from soweego.linker import train
 
+__author__ = 'Marco Fossati'
```

Comment:
The following 5 lines are misplaced.

Reply:
information moved to top of file in 2bb80a9d00839da174f982c93bbf57ab7349b23d